kolibri/core/assets/src/views/language-switcher/list.vue

```diff
@@ -140,4 +140,11 @@
   .language-list-dropdown
     margin-bottom: 8px
 
+  .language-list-dropdown.k-select-inline
+      width: auto
```

Comment:
Since we set width to auto here, you can keep `width: 100px` in the k-select component.

Reply:
Done adding it @christianmemije